tests/whitebox/integration/test_timeout.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2016 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Test 'timeout'.
+"""
+
+import unittest
+
+from stratis_cli._errors import StratisCliEnvironmentError
+
+from stratis_cli._actions._data import get_timeout
+
+
+class TimeoutTestCase(unittest.TestCase):
+    """
+    Test various timeout inputs.
+    """
+
+    def testTimeoutTooLarge(self):
+        """
+        Should fail because the timeout value is too large.
+        """
+        test_value = "2147483648"
```

Comment:
The whole body of the test can be condensed into:

```
with self.assertRaises(StratisCliEnvironmentError):
    get_timeout("2147483648")
```

which is more correct. The problem with the way your test is setup is that if an exception that is not the expected one is raised, the test framework sees a test error, not a test failure. Go ahead and update all the other similar tests, similarly.

Reply:
@waltdisgrace It looks like you're not using ```assertRaises``` where you should be.